public/javascripts/SVLabel/src/SVLabel/navigation/MapService.js

```diff
@@ -587,6 +587,15 @@ function MapService (canvas, neighborhoodModel, uiMap, params) {
         return uiMap.pano.find('svg').parent();
     }
 
+    /**
+     * Get link layer of bottom links
```

Comment:
Can you add a period at the end of the comment, as per [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide#general-guidelines). Later this summer I want to get our linter up and running. But until then, you get manual style warning messages from me :grin: